src/NUnitEngine/nunit.engine.tests/Runners/MasterTestRunnerTests.cs

```diff
@@ -166,7 +150,7 @@ public void CheckInternalRunner()
         {
             var prop = typeof(MasterTestRunner).GetField("_engineRunner", BindingFlags.NonPublic | BindingFlags.Instance);
             var runner = prop.GetValue(_runner);
-            Assert.That(runner, Is.TypeOf(_expectedRunner));
```

Comment:
We talked before about these tests overlapping with the RunnerSelection tests I've been working on. Looking through this class in a bit more detail, I think it's just this `CheckInternalRunner` method which is the only actual overlap. Once the RunnerSelection tests are all finished off, we could remove this method, and simplify this test class a little by taking the runner types out, and have it test one less thing. What do you think? 🙂

Reply:
That makes sense to me.